infrastructure/persistence/src/main/resources/db/migration/V9__change_user_id_index_to_unique.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE user_sil
+    DROP INDEX user_id;
+
+ALTER TABLE user_sil
+    ADD UNIQUE INDEX user_id_unique (user_id);
```

Comment:
p3) index naming table_column_index(uindex) 어떤가요?
```suggestion
    ADD UNIQUE INDEX user_sil_user_id_uindex (user_id);
```